### PR TITLE
feat: support getting actions

### LINKED ISSUE #91: support getting actions
Create action creator for getting actions and support it within the local API.

### PR DESCRIPTION
**Pull request by juancarlosfarah**
closes #91

### CODE REVIEW

**Comment by juancarlosfarah** [comment]
Hi everyone! So here I have added the functionality to get all of the actions of one or more spaces. I've included sample data for two spaces (you can imagine that these are two phases within the parent learning space). You are able to get this data, filtering for `spaceId`, `userId` and `visibility`. I have also included two sample calls:

1. Under `TeacherMode` where the teacher gets all the actions for all students.
2. Under `StudentMode` where a student only gets his/her data.

This should be a good starting point for your visualisation apps. Please review the code. As soon as it gets into `master`, it will be the default version when creating a app via the Graasp CLI.

**Comment by juancarlosfarah** [comment]
@ugGit, I add you to this PR so that you can get a bit more insight on (1) our PR workflow and (2) the starter app.

File: data/default.json [hunk 1]
```diff
@@ -60,6 +62,2854 @@
...
+        "ll": [46.516, 6.6328],
+        "metro": 0,
+        "area": 50
+      },
+      "createdAt": "2019-07-16T11:15:04.220Z",
+      "updatedAt": "2019-07-16T11:15:04.220Z",
+      "__v": 0
+    },
+    {
+      "_id": "5ca467b2bf8a9500322dc14e",
```
Comment (ugGit) [inline data/default.json:275]: @juancarlosfarah What is the benefit of an action without a verb?

Comment (juancarlosfarah) [reply data/default.json:275]: Actions don't require a `verb`. Often they have one, but if they don't, we will still store them.

**Comment by juancarlosfarah** [comment]
@HadiKhai @RolandSall @hasanagh, this is now in `master` and therefore available through the Graasp CLI.

**Pull request merged**